extensions/amp-riddle-quiz/validator-amp-riddle-quiz.protoascii

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-riddle-quiz
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+  name: "amp-riddle-quiz"
+    allowed_versions: "0.1"
+    allowed_versions: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-riddle-quiz>
```

Comment:
let's put the validator tests back so we can figure out why they were breaking in presubmit checks.

Reply:
how can i put the validator tests back?

Reply:
They were removed in this [commit](https://github.com/ampproject/amphtml/pull/12196/commits/a4bca46839cd23f6dc97f00db4481e4701712ff7) so you can grab them from there.